app/models/state_file_nj_intake.rb

```diff
@@ -84,6 +85,8 @@ class StateFileNjIntake < StateFileBaseIntake
 
   encrypts :account_number, :routing_number, :raw_direct_file_data, :raw_direct_file_intake_data
 
+  enum household_rent_own: { unfilled: 0, rent: 1, own: 2, neither: 3 }, _prefix: :household_rent_own
```

Comment:
[pebble[ I believe "both" is also supposed to be an option according to the designs? We should at least capture that here in the model (and it looks like an option on the XML as well?)

Reply:
Re: both: same response as the other design question - those are WIP so I implemented the case described by the ticket acceptance criteria and followed the version of the design that matched those.

Re: neither vs unfilled: all versions of the design seemed to have this option, so I assumed it meant that we didn't want to check anything by default, and did want the user to definitively answer the question.